Testscripts/Linux/CORE-InitrdModulesCheck.sh

```diff
@@ -26,6 +26,8 @@ SearchModules()
         abs_path="/root/initr/lib/modules/"
     elif [[ -d "/usr/lib64/modules" ]]; then
         abs_path="/usr/lib64/modules/"
+    elif [[ -d "/usr/lib/modules" ]]; then
+        abs_path="/usr/lib/modules/$(uname -r)"
```

Comment:
Can we add some comments here ?

Below observation is not in the scope of this PR
* It seems like this list may grow in future. We should find some concrete solution to handle all scenario for different distro

Reply:
@johnsongeorge-w Sure, thanks, I will include this in next PR.